src/wp-includes/js/media/controllers/cropper.js

```diff
@@ -24,16 +24,41 @@ Cropper = wp.media.controller.State.extend({
 		doCropArgs: {}
 	},
 
+	/**
+	 * @summary Opens crop image window.
+	 *
+	 * Shows the crop image window when called from the Add new image button.
+	 *
+	 * @since 4.2.0
+	 *
+	 * @returns void
+	 */
 	activate: function() {
 		this.frame.on( 'content:create:crop', this.createCropContent, this );
 		this.frame.on( 'close', this.removeCropper, this );
 		this.set('selection', new Backbone.Collection(this.frame._selection.single));
 	},
 
+	/**
+	 * @summary Changes the state of the toolbar window to browse mode.
+	 *
+	 * @since 4.2.0
+	 *
+	 * @returns void
+	 */
 	deactivate: function() {
 		this.frame.toolbar.mode('browse');
 	},
 
+	/**
+	 * @summary Creates crop image window.
+	 *
+	 * Initialized when clicking on the Select and Crop button.
```

Comment:
You might consider using the `@fires` attribute. This describes the event that triggers this method.